bridge_adaptivity/module/views.py

```diff
@@ -745,6 +743,23 @@ def callback_sequence_item_grade(request):
     return HttpResponse(xml, content_type="application/xml")
 
 
+def _check_and_build_web_socket_message(sequence, score):
+    """
+    Build a dictionary with data for callback by Web Socket.
+
+    Check flags ui_option, congratulation_message and validate score.
+    """
+    web_socket_message_dict = {}
```

Comment:
I would use defaultdict here.

Reply:
It seems no do some positive effects for this code if we would use `defaultdict`. Because I use the only 'True'  value of bool type. The default value of bool is `False`. It isn't suited for this part of the code.  This code creates the dictionary with `True` values for keys `is_button_enable` and  'is_show_pop_up'. And the last isn't created if a score is low.  The `ui_details` is not a bool value. This is a dictionary.